pkg/clusters/addons/kong/kong.go

```diff
@@ -29,6 +30,24 @@ const (
 	AddonName clusters.AddonName = "kong"
 
 	httpPort = 80
+
+	// DefaultEnterpriseImageRepo default kong enterprise image
+	DefaultEnterpriseImageRepo = "kong/kong-gateway"
+
+	// DefaultEnterpriseImageTag latest kong enterprise image tag
+	DefaultEnterpriseImageTag = "2.5.0.0-alpine"
+
+	// KongLicenseSecretName is the kong license data secret name
+	KongLicenseSecretName = "kong-enterprise-license"
+
+	// EnterpriseLicense is the kong enterprise test license
+	EnterpriseLicense = "KONG_ENTERPRISE_LICENSE"
+
+	// EnterpriseSuperUseerPwd is the super user password
+	EnterpriseSuperUseerPwd = "kong-enterprise-superuser-password"
```

Comment:
Typo, and I think it would be helpful to rename this var:

```suggestion

	// EnterpriseAdminPasswordSecretName is the name of the Kubernetes secret which provides
	// the admin auth for the Kong Admin API in enterprise deployments.
	EnterpriseAdminPasswordSecretName = "kong-enterprise-superuser-password"
```

Reply:
2257e7c